config/settings/production.yml

```diff
@@ -43,3 +43,26 @@
 # Social media connection information.
      facebook_app_id: <%= ENV['FACEBOOK_APP_ID'] %>
      share_progress_api_key: <%= ENV['SHARE_PROGRESS_API_KEY'] %>
+
+
+# Braintree
+     braintree:
+       merchant_id: <%= ENV['BRAINTREE_MERCHANT_ID'] %>
+       public_key: <%= ENV['BRAINTREE_PUBLIC_KEY'] %>
+       private_key: <%= ENV['BRAINTREE_PRIVATE_KEY'] %>
+
+       merchants:
+         EUR: <%= ENV['BRAINTREE_MERCHANT_ID_EUR'] %>
```

Comment:
I agree! It's getting insane. I think a general braintree yml file - one for sandbox, and another for production. Without our keys, the information will be meaningless to anyone else.